src/index.html

```diff
@@ -44,335 +43,472 @@
-                                error iure voluptate.</p>
+        <div class="values">
+            <div class="container">
+                <div class="row words">
+                    <div class="col-sm-6 col-md-4 universal">
+                        <div class="thumbnail">
+                            <img src="./images/infinity.svg" alt="inifinity">
+                            <div class="caption">
+                                <h3>UNIVERSAL</h3>
+                                <p>Sonar Planet has been designed to work on every blockchain. The community can contribute
+                                    to extend capabilities and to watch any kind of public keys.</p>
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="col-sm-6 col-md-4">
-                    <div class="thumbnail">
-                        <img src="" alt="">
-                        <div class="caption">
-                            <h3>NO-SIGNUP</h3>
-                            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Odio iusto hic repellat consectetur
-                                error iure voluptate.</p>
+                    <div class="col-sm-6 col-md-4 no-signup">
+                        <div class="thumbnail">
+                            <img src="./images/plus-circle.svg" alt="plus">
+                            <div class="caption">
+                                <h3>NO-SIGNUP</h3>
+                                <p>Sonar Planet is really simple to use. Add the public key to watch, select the kind of notification
+                                    you want and that's all.
+                                    <div>
+                                        GO AHEAD AND STAY TUNED!
+                                    </div>
+                                </p>
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="col-sm-6 col-md-4">
-                    <div class="thumbnail">
-                        <img src="" alt="">
-                        <div class="caption">
-                            <h3>SIMPLE</h3>
-                            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Odio iusto hic repellat consectetur
-                                error iure voluptate.</p>
+                    <div class="col-sm-6 col-md-4 simple">
+                        <div class="thumbnail">
+                            <img src="./images/lock.svg" alt="lock">
+                            <div class="caption">
+                                <h3>SIMPLE</h3>
+                                <p>
+                                    Sonar Planet is totally private, you choose your level of anonymity. No email and no phone required, unless you want to receive
+                                    your notification.
```

Comment:
I would change it by `unless you want to be notified via these channels`

Reply:
Done!